scram-tools.file/tools/tensorflow/tensorflow.xml

```diff
@@ -3,6 +3,7 @@
     <environment name="TENSORFLOW_BASE" default="@TOOL_ROOT@"/>
     <environment name="LIBDIR" default="$TENSORFLOW_BASE/lib"/>
     <environment name="INCLUDE" default="$TENSORFLOW_BASE/include"/>
+    <environment name="KERAS_BACKEND" default="tensorflow"/>
```

Comment:
it should be `<runtime ..../>` type variable ( see ROOTSYS as an example)
did you run test locally to see if gpu unit tests passed after setting this?

Reply:
I tested it by setting the environment variable manually (not via toolfile).

Reply:
for me all the unit tests still fails with error
```
##Failure Location unknown## : Error
Test name: testHelloWorldCUDA::test
uncaught exception of type std::exception (or derived).
- An exception of category 'UnavailableAccelerator' occurred while
   [0] Calling tensorflow::setBackend()
Exception Message:
Cuda backend requested, NVIDIA GPU visible to cmssw, but not visible to TensorF
low in the job
```

Reply:
Some tests that were failing previously worked after setting `KERAS_BACKEND`. Yes, I saw these failures as well - I thought I missed some setup step to make them work (in a container started with `--nv` flag)

Reply:
Could the failure be due to 12.4 not being an officially tested CUDA version for TF 2.16.1 (and even 2.17) - [link](https://www.tensorflow.org/install/source#gpu) lists 12.3 as officially tested version?

Reply:
are you sure you started `cmssw-el8` with `--nv` option? For me the following command
```
python3 -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"
```
 runs fine (both for this PR and TF_X Ibs) and return
```
[PhysicalDevice(name='/physical_device:GPU:0', device_type='GPU')]
```

Reply:
Now it works for me as well, weird.